fairlogger.sh

```diff
@@ -0,0 +1,46 @@
+package: FairLogger
+version: "%(tag_basename)s"
+tag: v1.1.0
+source: https://github.com/FairRootGroup/FairLogger
+build_requires:
+ - CMake
+ - "GCC-Toolchain:(?!osx)"
+incremental_recipe: |
+  cmake --build . --target install ${JOBS:+-- -j$JOBS}
+  mkdir -p $INSTALLROOT/etc/modulefiles && rsync -a --delete etc/modulefiles/ $INSTALLROOT/etc/modulefiles
+---
+mkdir -p $INSTALLROOT
+
+cmake $SOURCEDIR                                                 \
+      ${CXX_COMPILER:+-DCMAKE_CXX_COMPILER=$CXX_COMPILER}        \
+      ${CMAKE_BUILD_TYPE:+-DCMAKE_BUILD_TYPE=$CMAKE_BUILD_TYPE}  \
+      ${CXXSTD:+-DCMAKE_CXX_STANDARD=$CXXSTD}                    \
+      -DCMAKE_INSTALL_PREFIX=$INSTALLROOT                        \
+      -DDISABLE_COLOR=ON
+
+cmake --build . ${JOBS:+-- -j$JOBS}
+ctest ${JOBS:+-j$JOBS}
+cmake --build . --target install
+
+# Detect the lib dir, so we do not have to hardcode it into the modulefile
+LIBDIR=$(cmake -LA .. | grep CMAKE_INSTALL_LIBDIR | cut -d "=" -f2)
```

Comment:
I prefer convention over configuration. Can we have libdir set to `lib` and avoid this detection completely?

Reply:
The project uses the https://cmake.org/cmake/help/v3.0/module/GNUInstallDirs.html module to configure the lib dir. It can be different from platform to platform.

Reply:
In other recipes we use `-DCMAKE_INSTALL_LIBDIR` and set it to `lib`, just grep. Similarly, `-DCMAKE_INSTALL_BINDIR=bin` does the same for `bin`. I feel no need for this generality at directories level, honestly: I would like to keep our recipes stick to simple conventions (`lib` and `bin`). This is what I meant with convention over configuration. I am asking again to please change the recipe accordingly (that's what we do elsewhere).